### ISSUE TITLE
AIR 33 doesn't support Java 8 features for Android extensions

### ISSUE THREAD

**Issue by itlancer** [Gamua/Adobe-Runtime-Support#84]
## Problem Description
Android application cannot packaged if it use ANEs with Java 8 features such as lambda expressions, annotation etc.
It has been tested with different Windows 8.1/10 PCs for packaging with latests AIR 33.0.1.220 and AIR 33.0.1.228. For armv7 and armv8 build targets. Different 64-bit Java 8 versions was tried.
Compile options used in gradle for Android Studio project:
```
compileOptions {
    sourceCompatibility JavaVersion.VERSION_1_8
    targetCompatibility JavaVersion.VERSION_1_8
}
```
The same problem in all cases.
Patching adt (mentioned here https://github.com/Gamua/Adobe-Runtime-Support/issues/48 ) doesn't help.

Related issue: https://github.com/Gamua/Adobe-Runtime-Support/issues/48

## Steps to Reproduce
Create Java ANE for Android that uses some of Java 8 features. Lamda expressions for example in my case. Build ANE. Try to package Android application with apk-captive-runtime target.
Application example with sources for app and ANE with build scripts attached.
[java8_ane.zip](https://github.com/Gamua/Adobe-Runtime-Support/files/3534267/java8_ane.zip)

```
public void init() {
        Activity mMainActivity = mFreContext.getActivity();
        View mMainView = mMainActivity.getWindow().getDecorView();
        mMainView.setOnClickListener((View view) -> Toast.makeText(mMainActivity, "Java 8 lambda feature test", Toast.LENGTH_LONG).show());
    }
```

**Actual Result:**
Application cannot be packaged. Got error:
```
dx tool failed:Uncaught translation error: com.android.dx.cf.code.SimException: ERROR in com.test.ane.Java8ANE.init:()V: invalid opcode ba - invokedynamic requires --min-sdk-version >= 26 (currently 13)
1 error; aborting
```

**Expected Result:**
Application will be packaged with Java 8 features support .

## Known Workarounds
none

**Comment by ajwfrost** [comment]
FYI I think we have a fix or this - needs to be worked into the proper workflow, but we can look at adding the appropriate command line options and updating the other tools so that we support this.
AIR-199 internally..

**Comment by ajwfrost** [comment]
Hmm.. looks more complex than we first thought. We've got the workflow going without any errors from any of the Android tools but when we test this, it's throwing a bootstrap error. Needs further investigation...

**Comment by ajwfrost** [comment]
Fixed in 33.0.2.281 which is now available from https://airsdk.harman.com/download
Please confirm and close this issue if it works for you!
thanks

**Comment by itlancer** [comment]
For me now it works fine with AIR 33.0.2.281. Java 8 labmda feature works. A bit later will test with much more complex cases.
Thanks.
